fisher.lua

```diff
@@ -421,16 +428,31 @@ function check_incoming_chunk(id, original, modified, injected, blocked)
             if current.monster == false and fish:with('bite_id', current.bite_id) then
                 current.key = original:sub(21)
                 stats.bites = stats.bites + 1
-                local delay = fish:with('bite_id', current.bite_id).delay
-                message(2, 'catching fish in %d seconds':format(delay))
-                windower.send_command('wait %d; lua i fisher catch %d':format(delay, stats.casts))
+                local delay 
+				if settings.random then
+					if math.random() < 0.5 then
```

Comment:
Oops, nevermind, that would only give you a range of -0.5 to 0.5. You could always do `1.0 - math.random() * 2` which wouldn't give the same precision but in reality I doubt you could notice the difference.